### PR TITLE
[4.0] workflows mobile

### PR DESCRIPTION
**Pull request by brianteeman**
The class used in com_workflows (hidden-sm-down) was removed from bs between bs4 alpha and the final release

This PR correctly changes the class used to `d-none d-md-table-cell`
### Testing Instructions
Make sure the relevant columns are not displayed in mobile views

### CODE REVIEW

File: administrator/components/com_workflow/tmpl/stages/default.php [hunk 1]
```diff
@@ -62,25 +62,25 @@
 						</caption>
 						<thead>
 							<tr>
-								<td style="width:1%" class="text-center hidden-sm-down">
+								<td style="width:1%" class="text-center d-none d-md-table-cell ">
```
Comment (Quy) [inline administrator/components/com_workflow/tmpl/stages/default.php]:
```suggestion
								<td style="width:1%" class="text-center d-none d-md-table-cell">
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 577de9e8497ff1fc35f8bfc384c65f6bda99533a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26592">issues.joomla.org/tracker/joomla-cms/26592</a>.</sub>

**Comment by priyankaSahutekdi** [comment]
I have tested this item :white_check_mark: successfully on 577de9e8497ff1fc35f8bfc384c65f6bda99533a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26592">issues.joomla.org/tracker/joomla-cms/26592</a>.</sub>

**Comment by punambaravkar** [comment]
I have tested this item :white_check_mark: successfully on 577de9e8497ff1fc35f8bfc384c65f6bda99533a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26592">issues.joomla.org/tracker/joomla-cms/26592</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26592">issues.joomla.org/tracker/joomla-cms/26592</a>.</sub>

**Comment by wilsonge** [comment]
Conflicts please

**Comment by brianteeman** [comment]
@wilsonge where was the class "order" added

**Comment by wilsonge** [comment]
It wasn't? You removed it as it was unused I believe

**Comment by wilsonge** [comment]
https://github.com/joomla/joomla-cms/pull/26593 This PR

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**